sale_order_product_availability_inline/views/sale_views.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_order_form" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='order_line']/tree//field[@name='product_id']"
+                position="attributes"
+            >
+                <attribute name="context">{
+                        'so_product_stock_inline': True,
```

Comment:
IMHO, is it better to using python_dict(depends on base_view_inheritance_extension) to append context, other than overriding? This way keeps other modules' context working.

Reply:
Can you please do a PR against our branch for adding this?

Reply:
Done. Just did my first PR to OCA, I'm not a developer, correct me if wrong.

Reply:
I have just merged it. You can check if it works soon on runbot.